src/components/NavigationFolder.vue

```diff
@@ -133,6 +142,9 @@ export default {
 			folderStats: undefined,
 			loadingMarkAsRead: false,
 			clearingCache: false,
+			addInput: false,
```

Comment:
instead of three boolean props we can also have one of type string and assign something like `hidden`, `editing` and `saving`